cypress/e2e/with-users/devices/list.spec.ts

```diff
@@ -45,7 +45,7 @@ context("Device listing", () => {
       name: /Create and add to tag changes/,
     }).click();
 
-    cy.findByRole("button", { name: /Save changes/ }).click();
+    cy.findByRole("button", { name: /Save changed/ }).click();
```

Comment:
Whoops, that's actually a typo in `DeviceConfiguration.tsx` - it really should be `Save changes`